lerobot/common/policies/act/modeling_act.py

```diff
@@ -22,35 +23,39 @@
 from lerobot.common.policies.normalize import Normalize, Unnormalize
 
 
-class ACTPolicy(nn.Module):
+class ACTPolicy(nn.Module, PyTorchModelHubMixin):
     """
     Action Chunking Transformer Policy as per Learning Fine-Grained Bimanual Manipulation with Low-Cost
     Hardware (paper: https://arxiv.org/abs/2304.13705, code: https://github.com/tonyzhaozh/act)
     """
 
     name = "act"
 
-    def __init__(self, cfg: ACTConfig | None = None, dataset_stats=None):
+    def __init__(self, config: ACTConfig | None = None, dataset_stats=None):
```

Comment:
Actually, keeping `cfg` should be doable, at least with the latest version of `huggingface_hub`. You would have to specify how `ACTConfig` is encoded/decoded (see https://huggingface.co/docs/huggingface_hub/guides/integrations#advanced-usage). Also, it's possible to add metadata to the uploaded models, typically to tag them as "lerobot" and add a task type.

Reply:
Thanks @Wauplin!

Re. this comment., thanks that's handy. I suppose I don't feel enough of a need to change it now, but I'll keep it in mind in case the opportunity arises.

Re. your general offer for help. Thanks a lot! I'll let the others know in case they missed it.

Reply:
For sure! No need to update the current implementation :)